### ISSUE TITLE
Trade Search does not update the search link if no results are found

### ISSUE THREAD

**Issue by dcgaines** [cpieprzak/PoeAggregator#25]
When using the Poe Trade tab, the link to the search will only update if results are found, so I can't set up a live search through that unless there are currently listings for it

**Comment by cpieprzak** [comment]
Yeah. At some point I'll look in to how to get that to function. In the meantime, I suggest searching via a chrome browser and copy the trade url part into the search listing menu (ex: QdOZlGOCw).
